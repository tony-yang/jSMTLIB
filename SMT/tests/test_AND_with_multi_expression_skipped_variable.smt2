(set-logic QF_LRA)
(set-info :source | SMT-COMP'06 organizers |)
(set-info :smt-lib-version 2.0)
(set-info :category "check")
(set-info :status sat)
(set-info :notes |This benchmark is designed to check if the DP supports bignumbers.|)
(declare-fun x1 () Real)
(declare-fun x2 () Real)
(declare-fun x3 () Real)
(declare-fun x4 () Real)
(assert (and (> (+ (* 3 x1) (* 6 x4)) 10) (> (+ (* 9 x3) (* 10 x2)) 30) ))
(check-sat)
(exit)
