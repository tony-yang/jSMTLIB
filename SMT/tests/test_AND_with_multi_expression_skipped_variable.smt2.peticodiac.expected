# set_logic: QF_LRA
# :source: | SMT-COMP'06 organizers |
# :smt-lib-version: 2.0
# :category: "check"
# :status: sat
# :notes: |This benchmark is designed to check if the DP supports bignumbers.|
p cnf 4 2
c 3.0 0.0 0.0 6.0
b 4 >:10.0 NO_BOUND
c 0.0 10.0 9.0 0.0
b 5 >:30.0 NO_BOUND
